CMakeLists.txt

```diff
@@ -145,6 +145,7 @@ if (ENABLE_VECTOR_PREDICATE_SHORT_CIRCUIT)
   )
 endif()
 
+option(ENABLE_NETWORK_CLI "Allows use of the network cli" ON)
```

Comment:
Yes, I remembered. However, this was only because of Travis and I'd therefore prefer to keep this available by default. So I disabled it in the `.travis.yml` file.